graphics/libpano13/DETAILS

```diff
@@ -1,11 +1,11 @@
           MODULE=libpano13
-         VERSION=2.9.19
+         VERSION=2.9.21
           SOURCE=$MODULE-$VERSION.tar.gz
       SOURCE_URL=$SFORGE_URL/panotools/
-      SOURCE_VFY=sha256:037357383978341dea8f572a5d2a0876c5ab0a83dffda431bd393357e91d95a8
+      SOURCE_VFY=sha256:79e5a1452199305e2961462720ef5941152779c127c5b96fc340d2492e633590
         WEB_SITE=http://panotools.sourceforge.net
          ENTERED=20080913
```

Comment:
Just above this line insert: TYPE=cmake

Using that variable in DETAILS, provided any dependencies in DEPENDS follows cmakes  "-D" and no other twiddling needs doing allows the absence of a BUILD.